pages/news/2020-08-31-gsoc2020-macros.html

```diff
@@ -0,0 +1,50 @@
+title: Concluding GSoC 2020: Mixxx Macros
+author: xerus
+date: 2020-08-31 12:00:00
+comments: yes
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+I am Janek, a Software Engineering student who has been in touch with the community and using Mixxx on and off for about two years.
+Throughout the last 3 months I have worked on a new feature called "Mixxx Macros", inspired by [Serato Flip](https://serato.com/dj/pro/expansions/flip), as part of the [Google Summer of Code 2020](https://summerofcode.withgoogle.com/projects/#4641898733502464).
+It enables you to create edits of your tracks without having to dive into a DAW and modify the audio files!
+
+## Usage
+Let us have a look at how to use this new feature with a common use-case:  
+You want to create an intro for a track by playing a part of the breakdown twice, then jump to the beginning.
+Since the first drop is a bit bland, you additionally want to jump straight to the second drop after the buildup.
+
+#### Preparation
+  - Make sure quantize is enabled so everything is aligned to the beats
+  - Set hotcue 1 before and hotcue 2 after the part of the breakdown to loop (hotcue 2 is merely a marker for orientation) and hotcue 4 at the start of the second drop. You also check the position of the cue at the beginning of the track.
+
+#### Recording
```

Comment:
Let's not go overboard with headlines and lists. Too many headlines disrupt the reading flow and don't add value.
```suggestion

First, make sure quantize is enabled so everything is aligned to the beats, then you can start prepare your track.
For this macro, you need to set hotcue 1 before and hotcue 2 after the part of the breakdown to loop (hotcue 2 is merely a marker for orientation). 
Another hotcue (e.g. hotcue 4) needs to be set at the start of the second drop.
Also, check the position of the main cue and make sure that it's at the beginning of the track.

```